src/main/scala/com/lifeomic/fhirlib/v3/resources/Patient.scala

```diff
@@ -114,4 +107,19 @@ class Patient(override val id: Option[String],
     }
     return Some(Years.yearsBetween(birthDate.get, DateTime.now).getYears)
   }
+
+  private def getExtensionCoding(urlContains: String, systemContainsOpt: Option[String]): Option[Coding] = {
```

Comment:
How about moving this the the Resource base class so that it's available on all resources?

Reply:
Done